app/components/Editor.js

```diff
@@ -113,13 +117,40 @@ export default class Editor extends React.Component {
         this.redactor.insertion.insertHtml(replacement, 'start');
         this.redactor.selection.restoreMarkers();
       },
-      onReceiveNote: (_note) => {
-        // Empty
-      },
       clearUndoHistory: () => {
         // Called when switching notes to prevent history mixup.
         $R('#editor', 'module.buffer.clear');
       },
+      noteShouldBeRendered: async (note) => {
```

Comment:
I'm not entirely sure if this is the best route. The delegate calls setEditorText indiscriminately, but maybe that's how it should be. Then the individual editors decide whether to oblige. And setEditorText can definitely have side-effects.